core/domain/question_services.py

```diff
@@ -217,3 +218,24 @@ def update_question(
     updated_question = apply_change_list(question_id, change_list)
     _save_question(
         committer_id, updated_question, change_list, commit_message)
+
+
+def save_new_question(committer_id, question):
```

Comment:
Why is this created? Wasn't there already a _create_new_question function? Either remove that or this.

Reply:
I've removed this. Thanks